src/Okta.Sdk.IntegrationTests/AuthorizationServersScenarios.cs

```diff
@@ -0,0 +1,879 @@
+
+            var createdPolicy = await createdAuthorizationServer.CreatePolicyAsync(testPolicy);
+            var updatedPolicy = await createdAuthorizationServer.UpdatePolicyAsync(testUpdatedPolicy, createdPolicy.Id);
+            try
+            {
+                updatedPolicy.Should().NotBeNull();
+                updatedPolicy.Name.Should().Be(testUpdatedPolicy.Name);
+            }
+            finally
+            {
+                await createdPolicy.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdPolicy.Id, createdPolicy.Id);
+                await createdAuthorizationServer.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerAsync(createdPolicy.Id);
+            }
+        }
+
+        [Fact(Skip = "OKTA-303121: Api validation failed: com.saasure.framework.exception.KeyStoreLimitExceededException (400, E0000001): Unable to create new authorization server. You cannot create authorization servers because signing keys could not be generated. To fix this issue, contact support.")]
+        public async Task DeleteAuthorizationServerPolicy()
+        {
+            var testClient = TestClient.Create();
+            var testAuthorizationServerName = $"{SdkPrefix}:Test AuthZ Server ({TestClient.RandomString(4)})";
+
+            var testAuthorizationServer = new AuthorizationServer
+            {
+                Name = testAuthorizationServerName,
+                Description = "Test Authorization Server",
+                Audiences = new string[] { "api://default" },
+            };
+            var testPolicy = new OAuthAuthorizationPolicy
+            {
+                Name = $"{SdkPrefix}:{nameof(DeleteAuthorizationServerPolicy)}:Test Policy",
+            };
+
+            var createdAuthorizationServer = await testClient.AuthorizationServers.CreateAuthorizationServerAsync(testAuthorizationServer);
+            var createdPolicy = await createdAuthorizationServer.CreatePolicyAsync(testPolicy);
+            try
+            {
+                var retrievedAuthorizationPolicy = await testClient.AuthorizationServers.GetAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdPolicy.Id);
+                retrievedAuthorizationPolicy.Should().NotBeNull();
+
+                await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdPolicy.Id);
+                var ex = await Assert.ThrowsAsync<OktaApiException>(() => testClient.AuthorizationServers.GetAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdPolicy.Id));
+                ex.StatusCode.Should().Be(404);
+            }
+            finally
+            {
+                await createdPolicy.DeactivateAsync();
+                await testClient.AuthorizationServers.DeleteAuthorizationServerPolicyAsync(createdAuthorizationServer.Id, createdPolicy.Id);
```

Comment:
I think this will fail since you already deleted the policy above.

Reply:
Good point.